src/server/services/vergunningen.ts

```diff
@@ -314,19 +353,8 @@ export function createVergunningNotification(
     const allItems = items.filter(
       (caseItem: Vergunning): caseItem is GPK => caseItem.caseType === 'GPK'
     );
-    let formLink = '';
-    switch (item.caseType) {
-      case 'GPK':
-        formLink =
-          'https://formulieren.amsterdam.nl/TripleForms/DirectRegelen/formulier/nl-NL/evAmsterdam/GehandicaptenParkeerKaartAanvraag.aspx/Inleiding';
-        break;
-      case 'Parkeerontheffingen Blauwe zone bedrijven':
-        formLink = ''; //Missing information from RD;
-        break;
-      case 'Parkeerontheffingen Blauwe zone particulieren':
-        formLink = ''; //Missing information from RD;
-        break;
-    }
+
+    const notificationLink = notificationLinks[item.caseType];
```

Comment:
Add a fallback here to the Vergunningen overzicht pagina so we never show a dead link.

Reply:
Or to the detail page of the caseitem?